### ISSUE TITLE
Cannot authentic after upgrade

### ISSUE THREAD

**Issue by rhyolight** [dennisreimann/ioctocat#429]
After purchasing an upgrade to the application I could no longer authenticate. I couldn't authenticate through the normal means or using the enterprise version.

**Comment by dennisreimann** [comment]
Can you try it again during the next hour? This might be related to APi outage or GitHub deploying etc.

**Comment by rhyolight** [comment]
Just tried, same auth failure.

**Comment by dennisreimann** [comment]
Okay, and you made sure you haven't mistyped the password or copied a whitespace or something like that?
I don't like to ask that, because I can image everyone to make sure this didn't happen, but in most cases when this resolves, it turns out that c&p with whitespaces etc. make this go wrong...

**Comment by rhyolight** [comment]
I was using the right password, I copied it from 1password (nice feature).
I fixed the problem, here's how:
- uninstalled the app completely
- reinstalled the app from the app store
- authenticated
- turned on the push notifications (required auth again, but failed at that
  point)
- exited and re-entered app settings page, tried push notifications again,
  and it worked.

Not sure what was going on, but it is working now.

 \m/ (>.<) \m/
matthew taylor
